sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -747,9 +753,8 @@ class SdkSynchronizer private constructor(
         }
 
     override suspend fun importAccountByUfvk(setup: AccountImportSetup): Account {
-        // TODO [#1772]: redirect to correct service mode after 2.1 release
         val chainTip: BlockHeight? =
-            when (val response = processor.downloader.getLatestBlockHeight(serviceMode = ServiceMode.Direct)) {
+            when (val response = processor.downloader.getLatestBlockHeight(serviceMode = ServiceMode.UniqueTor)) {
```

Comment:
Found another spot without the necessary flag check:
```suggestion
            when (val response = processor.downloader.getLatestBlockHeight(serviceMode = sdkFlags ifTor ServiceMode.UniqueTor)) {
```